icefoganalytics/elcc-data-management
# ELCC-7: Make Admin and Quality section mirror data input in Spaces

Fixes ELCC-7

# Context

The user enters data into the Spaces section, and those numbers should carry-over into the Admin and Quality sections.  The user should still be able to edit the Admin and Quality numbers.

To give more background info on that, Spaces and Admin can go over the number of licensed spots, however we will only pay up to the spots amount for Quality.

![image](https://github.com/icefoganalytics/elcc-data-management/assets/23045206/1c3b53dc-8d4a-4c28-a8af-cde6bca728ad)

# Implementation

Fixed a bunch of snake -> camel casing issues throughout the app.
Migrated some json data stored in snake case.

# Screenshots

(if making UI changes add screenshots comparing the old and new experience)

# Testing Instructions

1. Boot the app via `dev up`

2. Check that the app complies, and that you can log in at http://localhost:8080.
3. Go to the /dashboard.
4. Click on the child care centre widget (graducation hat)
5. Click on a daycare.
6. Click on the “view details” tab on the far left widget thing.
7. Then click the “Worksheets” tab, and click on “add worksheets for 2023/24”
8. Wait for the notification “fiscal year added”.
9. TODO
